history/history-network.md

```diff
@@ -315,8 +315,13 @@ An `Offer` message containing ephemeral headers **MUST**
 * Ephemeral header content keys offered to the client **MUST** be in a strictly consecutive descending order by `block_number`
 * An `Offer` message can only contain up to 31 ephemeral header content keys
 
-Bridges **MUST** send `Offer` messages that
-* Contain content keys from the latest tip as specified by `LightClientUpdates` to the common ancestor block if a re-org occurred. So normally this would be 1 header, unless a re-org occurred where multiple would be included. An additional `8` header's content keys, MUST be included in the `Offer` message as padding. If bridges detect a re-org + padding depth is larger then 31 content keys, additional `Offer` messages containing the rest of the chain should be sent by the bridge in 10 second intervals of the initial `Offer` message
+Bridges **MUST** send `Offer` messages that contain content keys of three conceptual categories: new, re-org, and padding
+* New headers are defined by the latest tip as specified by `LightClientUpdates`, which have not been broadcast before
+* If a re-org occurred, then the headers back to the common ancestor block from a previous offer are included
+* An additional `8` header's content keys, MUST be included in the `Offer` message as padding
+* Often, bridge updates will inclued 1 new header, 0 re-org headers, and 8 padding headers
```

Comment:
I am not sure there is a reason to specify `have not been broadcast before`, that might limit implementation creativity

```suggestion
Bridges **MUST** send `Offer` messages that
* Contain content keys for the latest header as specified by `LightClientUpdates`, headers corresponding to the re-orged depth, and 8 additional headers which serve as padding.
* The common case would be 1 new header, 0 re-orged headers, and 8 headers for padding
```

I want to stop using `common ancestor block` terminally thinking back on it, I think it is fairly confusing, going forward I would prefer if we use `re-orged depth` or `re-org depth`  instead.

Reply:
Yeah, using `re-org depth` sounds good. Especially if we could add somewhere `1 + re-org depth + 8`, as I find that this explains it very well.